views/calendar/types.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type momentT from 'moment'
+
+export type EventType = {
```

Comment:
This is the shape of the cleaned data that make after processing the data from Google Calendar. Raw data is GoogleCalendarEventType.